src/app/detail/ui/NewTaskCard.tsx

```diff
@@ -53,6 +54,8 @@ export const NewTaskCard = ({
   handleSubTaskCreation: (payload: CreateTaskRequest) => void
 }) => {
   const { workflowStates, assignee, token, filterOptions, previewMode, activeTask } = useSelector(selectTaskBoard)
+  const { assigneeListForLimitedTasks } = useSelector(selectTaskDetails)
```

Comment:
```suggestion
  const { activeTaskAssignees } = useSelector(selectTaskDetails)
```
Might be a better name